reactnative/RNPlayground/app/screens/AboutScreen.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react"
+import { observer } from "mobx-react-lite"
```

Comment:
What is an observer on the context of React?

Reply:
This `observer` comes from `MobX`, which works independantly from React - but often used together. Its a simple, scalable, state-management lib that helps us make React native reactive - as these aren't reactive out the box.

The gist of MobX; https://mobx.js.org/the-gist-of-mobx.html

![image](https://user-images.githubusercontent.com/31416565/221875856-7a21e39a-27b2-46f8-b3b3-8dc1aaa75f7c.png)